easybuild/framework/easyconfig/default.py

```diff
@@ -165,6 +165,7 @@
 
     # OTHER easyconfig parameters
     'buildstats': [None, "A list of dicts with build statistics", OTHER],
+    'options' : [{},"A list of extra options for specific blocks", OTHER],
```

Comment:
The `options` thing is not a generic way of providing *anything* to easyblock, only certain things are picked up there (and any others are blatently ignored). In particular, for something installed as a separate module, probably only `modulename` is picked up from `self.cfg['options']` (although I'd have to check)...

So, do we need something else/better? Yes, we do. See also https://github.com/hpcugent/easybuild-easyconfigs/pull/1349#discussion-diff-23835695 .

The `options` thing is there for extensions, where custom `source_urls`, `patches`, etc. can be specified for a particular extension (see https://github.com/hpcugent/easybuild-framework/blob/master/easybuild/framework/easyblock.py#L387).

Ideally, we need to simply make *all* easyconfig parameters available to extensions, not just a couple (let alone custom ones).